core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/listener/UserManagementErrorEventListener.java

```diff
@@ -398,4 +399,6 @@ default boolean onListUsersFailure(String errorCode, String errorMessage, String
         return true;
     }
 
+    boolean onGetUserListFailure(String errorCode, String errorMessage, String claim, String claimValue,
```

Comment:
make this a default method to avoid api change in a patch release.
The name seems to be not matching the intended functionality.